quarkus/config-api/src/main/java/org/keycloak/config/BootstrapAdminOptions.java

```diff
@@ -1,35 +1,39 @@
 package org.keycloak.config;
 
 public class BootstrapAdminOptions {
+    
+    public static final String DEFAULT_TEMP_ADMIN_USERNAME = "temp-admin";
+    public static final String DEFAULT_TEMP_ADMIN_SERVICE = DEFAULT_TEMP_ADMIN_USERNAME;
+    public static final int DEFAULT_TEMP_ADMIN_EXPIRATION = 120;
```

Comment:
Does it have to be exposed now? IMO, it might be just commented out. And for the future, it'd be also good to know the units of the expiration value - rather being more explicit.

Reply:
It is not exposed, it was just moved from the appliancebootstrap.

Reply:
yep, right, I thought more like it's visible even when we do not have any expiration yet :P 

But it's a nitpick. LGTM